packages/core/contracts/financial-templates/contract-for-difference/ContractForDifference.sol

```diff
@@ -114,30 +114,30 @@ contract ContractForDifference is Testable, Lockable {
         uint64 _expirationTimestamp,
         uint256 _collateralPerPair,
         bytes32 _priceIdentifier,
-        ExpandedIERC20 _longTokenAddress,
```

Comment:
curious what you guys think about this change. Including `Address` in the variable name is not very useful so I stripped it out. However, this might be a bit inconsistent with other variables in other contracts. For example, the `PricelessPositionManager` calls addresses `_finderAddress` etc. curious what you guys think is a good general rule.

Reply:
I think you're ok not following the naming convention of the `PPM`, as long as the naming convention in this contract is similar.